REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.3
 Polynomials
 Compat 0.4.1
+ProgressMeter
```

Comment:
This is very odd to introduce into this PR and introduces a new (albeit small) dependency. Shouldn't this be own own PR?